plugins/inputs/internet_speed/internet_speed.go

```diff
@@ -92,6 +100,7 @@ func (is *InternetSpeed) Gather(acc telegraf.Accumulator) error {
 		if err != nil {
 			return fmt.Errorf("upload test failed failed: %w", err)
 		}
+		pLoss, _ = analyzer.RunMulti(is.servers.Hosts())
```

Comment:
Can you please add a comment that the error will always be `nil` or check the error?

Reply:
ok, warning packet loss not applicable here.